### ISSUE TITLE
Retrain KASH nuc and actin models

### ISSUE THREAD

**Issue by ninanikitina** [mal-boisestate/afilament#43]
MODEL ACTIN

SET:
32 img of 1 initial cell by Zeiss 63x - Alexa488, no Adipo
10 img of 1 cell by Leica 63 x - Alexa488, no Adipo

Weight correction = 200, pos_weight = torch.tensor(200).to(device=device)

INFO: Starting training:
        Epochs:          200
        Batch size:      1
        Learning rate:   0.001
        Training size:   38
        Validation size: 6
        Checkpoints:     True
        Device:          cuda
        Images scaling:  1.0

LOG: Mar16_19-23-06_LAPTOP-ISSFJ09KLR_0.001_BS_1_SCALE_1.0
